pkgs/misc/uboot/default.nix

```diff
@@ -606,6 +607,20 @@ in
     filesToInstall = [ "u-boot.rom" ];
   };
 
+  ubootQuartz64B = buildUBoot {
+    defconfig = "quartz64-b-rk3566_defconfig";
+    extraMeta.platforms = [ "aarch64-linux" ];
+    BL31 = "${armTrustedFirmwareRK3568}/bl31.elf";
+    ROCKCHIP_TPL = rkbin.TPL_RK3568;
```

Comment:
So TPL_RK3566 is not used after all?

Reply:
Oops. Thanks for reporting it. Fixing it with #402958.

Reply:
They're the same.

```
~/.../bin/rk35 $ md5sum rk356*_ddr_{920,1056}MHz_v1.23.bin | sort -u
5db650a1c97fd572bdc96e5218fb9c02  rk3566_ddr_920MHz_v1.23.bin
5db650a1c97fd572bdc96e5218fb9c02  rk3568_ddr_920MHz_v1.23.bin
9aef462eee359e8cf7ac1367dfd0f8d8  rk3566_ddr_1056MHz_v1.23.bin
9aef462eee359e8cf7ac1367dfd0f8d8  rk3568_ddr_1056MHz_v1.23.bin
```

The RK3566 and RK3568 are largely the same, except for some peripherals.